src/robot/parsing/lexer/tokens.py

```diff
@@ -143,7 +143,7 @@ def __init__(self, type=None, value=None, lineno=-1, col_offset=-1, error=None):
             value = {
                 Token.IF: 'IF', Token.ELSE_IF: 'ELSE IF', Token.ELSE: 'ELSE',
                 Token.FOR: 'FOR', Token.END: 'END', Token.CONTINUATION: '...',
-                Token.EOL: '\n'
+                Token.EOL: '\n', Token.WITH_NAME: 'WITH NAME'
```

Comment:
Tokens automatically getting the value are listed in the class doc string and this should be added there as well. Could also consider adding a class attribute like `AUTO_VALUE = {...}` that could be referenced from the docs instead.

Reply:
I added missing token to docs (manually). `AUTO_VALUE` class attribute can be added separetly. If we're going to add it, should it be also used in `__init__`? ie:
```
class Token:
    """
    :attr:`AUTO_VALUE`
    """
    AUTO_VALUE = {
                Token.IF: 'IF', Token.ELSE_IF: 'ELSE IF', Token.ELSE: 'ELSE',
                Token.FOR: 'FOR', Token.END: 'END', Token.CONTINUATION: '...',
                Token.EOL: '\n', Token.WITH_NAME: 'WITH NAME'
    }
    
    def __init__(self, value=None, ...):
        if value is None:
            value = self.AUTO_VALUE.get(type, '')

Reply:
Yeah, if we'd add AUTO_VALUE it should work like that. Probably in `__init__` like
```
if value is None and type in self.AUTO_VALUE:
    value = self.AUTO_VALUE[type]
```
Probably not worth the effort at least now.